doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -77,9 +77,10 @@ Short answer
    linear algebra operations.
 -  You can have standard vectors or row/column vectors if you like.
 
-The only disadvantage of using the array type is that you will have to
-use ``dot`` instead of ``*`` to multiply (reduce) two tensors (scalar
-product, matrix vector multiplication etc.).
+Until Python 3.5 the only disadvantage of using the array type is that you
```

Comment:
All looks good, except that I'd have this sentence in past tense: ... array type *was* that you *had* to ...